sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/pipeline/_io/base.py

```diff
@@ -348,6 +385,7 @@ def __init__(
         *,
         data: Optional[Union[Output, str]] = None,
         owner: Optional[Union["BaseComponent", "PipelineJob"]] = None,
+        node_output: Optional["NodeOutput"] = None,
```

Comment:
This name is not very intuitive for me - it's kind of confusing to have a node output for a pipeline output. One proposal: `binding_output`.

Feel free to resolve this comment as it's about naming only.

Reply:
Updated